server_admin/topics/authentication/webauthn.adoc

```diff
@@ -59,37 +55,31 @@ If a user does not have WebAuthn credentials, the user must register WebAuthn cr
 Users can log in with WebAuthn if they have a WebAuthn credential registered only. So instead of adding the *WebAuthn Authenticator* execution, you can:
 
 .Procedure
-. Click on the _Actions_ link for *WebAuthn Browser Forms*.
-. Click *Add flow*.
-. Enter "Conditional 2FA" for the _Alias_ field.
-. Click *Save*.
-. Click _CONDITIONAL_ for *Conditional 2FA*
-. Click on the _Actions_ link for *Conditional 2FA*.
-. Click *Add execution*.
-. Click the *Provider* drop-down list.
-. Click *Condition - User Configured*.
-. Click *Save*.
-. Click _REQUIRED_ for *Conditional 2FA*
-. Click on the _Actions_ link for *Conditional 2FA*.
-. Click *Add execution*.
-. Click the *Provider* drop-down list.
-. Click *WebAuthn Authenticator*.
-. Click *Save*.
-. Click _ALTERNATIVE_ for *Conditional 2FA*
+. Click *+* menu of the *WebAuthn Browser Forms* row.
+. Click *Add sub-flow*.
+. Enter "Conditional 2FA" for the _name_ field.
+. Click *Add*.
+. Select *Conditional* for the *Conditional 2FA* to set its requirement to conditional.
+. Click *+* menu of the *Conditional 2FA* row.
+. Click *Add condition*.
```

Comment:
On the *Conditional 2FA* row, click the plus sign *+* and select *Add condition*.